v1/api/sites.ts

```diff
@@ -104,17 +108,18 @@ export const siteRoutes = (store: StoreI) => async (server: FastifyTypebox): Pro
 
     await store.sites.delete(id)
     await store.publisher.unregisterSiteFromAllPublishers(id)
+    await store.fs.clear(id)
     return await reply.send()
   })
 
   server.post<{
     Params: {
       id: string
     }
-    Body: Static<typeof UpdateSite>
+    Body: Static<typeof ProtocolStatus>
   }>('/sites/:id', {
     schema: {
-      body: UpdateSite,
+      body: ProtocolStatus,
```

Comment:
Mind wrapping this in one more layer so that we can add more fields that can be added in the future? .e.g `Body: { protocols: ProtocolStatus}`